catalog/ui/search-ui/simple/src/main/java/org/codice/ddf/ui/searchui/simple/properties/UiConfigurationPropertiesFactory.java

```diff
@@ -15,84 +15,48 @@
 
 import java.util.Optional;
 import org.codice.ddf.branding.BrandingRegistry;
+import org.osgi.service.cm.ConfigurationAdmin;
 
 /** Stores external configuration properties. */
```

Comment:
✏️  This javadoc should probably be updated or removed.

Reply:
I vote for updated 😄